contracts/BSCValidatorSet.sol

```diff
@@ -153,38 +130,38 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
 
     /*----------------- events -----------------*/
     event validatorSetUpdated();
-    event validatorJailed(address indexed validator);
-    event validatorEmptyJailed(address indexed validator);
-    event batchTransfer(uint256 amount);
-    event batchTransferFailed(uint256 indexed amount, string reason);
-    event batchTransferLowerFailed(uint256 indexed amount, bytes reason);
     event systemTransfer(uint256 amount);
-    event directTransfer(address payable indexed validator, uint256 amount);
-    event directTransferFail(address payable indexed validator, uint256 amount);
     event deprecatedDeposit(address indexed validator, uint256 amount);
     event validatorDeposit(address indexed validator, uint256 amount);
     event validatorMisdemeanor(address indexed validator, uint256 amount);
     event validatorFelony(address indexed validator, uint256 amount);
-    event failReasonWithStr(string message);
-    event unexpectedPackage(uint8 channelId, bytes msgBytes);
     event paramChange(string key, bytes value);
     event feeBurned(uint256 amount);
     event validatorEnterMaintenance(address indexed validator);
     event validatorExitMaintenance(address indexed validator);
     event finalityRewardDeposit(address indexed validator, uint256 amount);
     event deprecatedFinalityRewardDeposit(address indexed validator, uint256 amount);
-    event tmpValidatorSetUpdated(uint256 validatorsNum);
+
+    event validatorJailed(address indexed validator);  // @dev deprecated
+    event validatorEmptyJailed(address indexed validator);  // @dev deprecated
+    event batchTransfer(uint256 amount);  // @dev deprecated
+    event batchTransferFailed(uint256 indexed amount, string reason);  // @dev deprecated
+    event batchTransferLowerFailed(uint256 indexed amount, bytes reason);  // @dev deprecated
+    event directTransfer(address payable indexed validator, uint256 amount);  // @dev deprecated
+    event directTransferFail(address payable indexed validator, uint256 amount);  // @dev deprecated
+    event failReasonWithStr(string message);  // @dev deprecated
+    event unexpectedPackage(uint8 channelId, bytes msgBytes);  // @dev deprecated
+    event tmpValidatorSetUpdated(uint256 validatorsNum);  // @dev deprecated
 
     /*----------------- init -----------------*/
     function init() external onlyNotInit {
-        (IbcValidatorSetPackage memory validatorSetPkg, bool valid) =
-            decodeValidatorSetSynPackage(INIT_VALIDATORSET_BYTES);
+        (ValidatorSetPackage memory validatorSetPkg, bool valid) =
+            decodeValidatorSet(INIT_VALIDATORSET_BYTES);
         require(valid, "failed to parse init validatorSet");
         for (uint256 i; i < validatorSetPkg.validatorSet.length; ++i) {
             currentValidatorSet.push(validatorSetPkg.validatorSet[i]);
             currentValidatorSetMap[validatorSetPkg.validatorSet[i].consensusAddress] = i + 1;
         }
-        expireTimeSecondGap = EXPIRE_TIME_SECOND_GAP;
```

Comment:
Should we change the `init` function? I think it's better to keep it in this way:
```
expireTimeSecondGap = 1000;
```